bindings/src/test/java/io/pravega/server/host/ExtendedS3IntegrationTest.java

```diff
@@ -117,7 +117,7 @@ public Storage createStorageAdapter() {
                     .withInitialRetryDelay(1)
                     .withProperty("com.sun.jersey.client.property.connectTimeout", 100);
 
-            S3JerseyClient client = new S3ClientWrapper(s3Config, filesystemS3);
+            S3JerseyClient client = new io.pravega.segmentstore.server.host.S3ClientWrapper(s3Config, filesystemS3);
```

Comment:
Isn't this class being also moved under the same package, is this change necessary?

Reply:
This is not necessary. Fixed.